### PR TITLE
[parser&tests] Move code parts

### PR DESCRIPTION
**Pull request by redref**

### CODE REVIEW

File: pkr/kard.py [hunk 1]
```diff
@@ -103,33 +93,62 @@ def list(cls, kubernetes=False):
...
 
     @classmethod
-    def create(cls, kard_name, env_name, driver_name, extra):
+    def create(cls, name, env, driver, features, meta, extra, **kwargs):
         """Factory method to create a new kard"""
+        extras = {"features": []}
+        if meta:
+            extras.update(yaml.safe_load(meta))
+        extras.update({a[0]: a[1] for a in [a.split("=", 1) for a in extra]})
+        for feature in features_merge(extras["features"]):
```
Comment (Kimamisa) [inline pkr/kard.py:102]: features_merge function name is not explicit. You should rename `features_merge`.

File: pkr/utils.py [hunk 2]
```diff
@@ -112,12 +112,36 @@ def merge(source, destination):
...
                 destination[key] = value
-        else:
+        elif overwrite or key not in destination:
             destination[key] = value
 
     return destination
 
 
+def diff(previous, current):
+    result = {}
```
Comment (Kimamisa) [inline pkr/utils.py:122]: maybe with a small docstring ?

File: pkr/kard.py (hunk 1 above)
Comment (redref) [reply pkr/kard.py:102]: Same name right ?

File: pkr/utils.py (hunk 2 above)
Comment (redref) [reply pkr/utils.py:127]: Done.

File: pkr/kard.py (hunk 1 above)
Comment (redref) [reply pkr/kard.py]: Handled

Comment (Kimamisa) [reply pkr/kard.py]: Great !

**Review by Kimamisa — APPROVED** [review]

**Pull request merged**